pavement.py

```diff
@@ -119,5 +119,5 @@ def release():
     sh("git tag -s '%(name)s-%(version)s' -m 'Version bump to %(version)s'" % {'name': NAME, 'version': VERSION})
     sh("git push --tags")
     sh("git push")
-    sh("python setup.py register sdist upload")
-    sh("python setup.py bdist_wheel upload")
+    sh("python setup.py register sdist bdist_wheel")
+    sh("twine upload dist/*")
```

Comment:
Maybe a better idea to make this `dist/*{version}*` so we don't re-upload old versions of mistakenly upload wrong versions.